pof/src/types.rs

```diff
@@ -2423,7 +2461,9 @@ pub enum Warning {
     SpecialPointPropertiesTooLong(usize),
     // path with no parent
     // thruster with no engine subsys (and an engine subsys exists)
-    // turret uvec != turret normal
+    NoFvec(ObjectId),
+    NoUvec(ObjectId),
+    UvecFvecNotPerpendicular(ObjectId),
```

Comment:
Preface all of these with 'Turret' or maybe 'TurretObject' so they're a little clearer when put alongside all the other warnings.